bundles/org.openhab.binding.tibber/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -185,6 +185,19 @@
 				<description>Current power consumption (as positive value) and production (as negative value)</description>
 			</add-channel>
 		</instruction-set>
+
+		<instruction-set targetVersion="6">
+			<add-channel id="total-price" groupIds="price">
+				<type>tibber:price</type>
+				<label>Total Price</label>
+				<description>Total price including energy and taxes</description>
+			</add-channel>
+			<add-channel id="tax" groupIds="price">
```

Comment:
Can you add an update channel for the spot-price, changin it to the new type _and_ setting the description to the new one with deprecated message?

Reply:
done